source/installation/restore-minio.rst

```diff
@@ -0,0 +1,274 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use :mc-cmd:`mc admin console` command *or* ``journalctl -u minio`` for
+``systemd``-managed installations to monitor the server log output after remounting
+drives. The output should include messages identifying each formatted and empty
+drive.
+
+Use :mc-cmd:`mc admin heal` to monitor the overall healing status on the
+deployment. MinIO aggressively heals replaced drive(s) to ensure rapid recovery
+from the degraded state.
+
+7) Next Steps
+~~~~~~~~~~~~~
+
+Monitor the cluster for any further drive failures. Some drive batches may fail
+in close proximity to each other. Deployments seeing higher than expected drive
+failure rates should schedule dedicated maintenance around replacing the known
+bad batch. Consider using |subnet| to coordinate with MinIO engineering around
+guidance for any such operations.
+
+.. _minio-restore-hardware-failure-node:
+
+Node Failure Recovery
+---------------------
+
+If a MinIO node suffers complete hardware failure (e.g. loss of all drives,
+data, etc.), the node begins healing operations once it rejoins the deployment.
+
+The procedure can be summarized as follows:
+
+1. Bring up the replacement node, ensuring it meets or exceeds the configuration
+   of the failed node.
+2. Update DNS to ensure the hostname for the failed node points at the
+   replacement node.
+3. Start the MinIO process on the new node ensuring the settings match all other
+   nodes in the deployment. The MinIO binary **must** match across all nodes.
+4. Check MinIO logs to verify the node has reconnected.
+5. Use :mc-cmd:`mc admin heal` to confirm the healing status.
+
+The replacement node hardware should be substantially similar to the failed
+node. There are no negative performance implications to using improved compute
+hardware.
+
+The replacement drive hardware should be substantially similar to the failed
+drive. For example, replace a failed SSD with another SSD drive of the same
+capacity. While you can use drives with larger capacity, MinIO uses the
+*smallest* drive's capacity as the ceiling for all drives in the :ref:`Server
+Pool <minio-intro-server-pool>`.
+
+The following steps provide a more detailed walkthrough of node replacement.
+These steps assume a MinIO deployment where each node has a DNS hostname 
+as per the :ref:`documented prerequisites <minio-installation>`.
+
+1) Start up the new node hardware
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ensure the new node has received all necessary security, firmware, and OS
+updates as per industry, regulatory, or organizational standards and
+requirements.
+
+The new node software configuration should closely match that of the other
+nodes in the deployment, including but not limited to the OS and Kernel 
+versions and configurations. Heterogeneous software configurations may result
+in unexpected or undesired behavior in the deployment.
+
+2) Update DNS to for the new host
```

Comment:
As above, changing DNS is not strictly required, and in fact is undesirable if you were changing hostname instead of ip (since it is then assumed you don't have contiguous names).  To avoid confusion we might remove reference to DNS and say something along the lines of setting the proper hostname and IP.

Reply:
Addressed